### PR TITLE
Adding image diffs for components

### PR DESCRIPTION
**Pull request by ironsidevsquincy**

### CODE REVIEW

**Comment by matthew-andrews** [comment]
These look great!

https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/article/2015-06-23/11:37-2114/successes/index.html

File: test/visual/test.js [hunk 1]
```diff
@@ -23,6 +23,34 @@ visualRegressionTest({
 			elements: {
 				article_body: '.article__body'
 			}
+		},
+		slideshow: {
+			path: "/7f473d26-be68-11e4-8036-00144feab7de",
+			widths: [500, 1400],
+			elements: {
+				article_body: '.article__gallery'
```
Comment (matthew-andrews) [inline test/visual/test.js]: ahh i just realised this won't work because slideshows are JS only atm…

Comment (matthew-andrews) [inline test/visual/test.js]: As in…… there's not even a non-js fallback :see_no_evil:

Comment (ironsidevsquincy) [inline test/visual/test.js]: oh, javascript doesn't run?

Comment (matthew-andrews) [inline test/visual/test.js]: at the minute yus

**Pull request merged**